tests/host_tools/fillmem.c

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+#define _GNU_SOURCE
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <errno.h>
+#include <sys/types.h>
+#include <sys/wait.h>
+#include <sys/stat.h>
+#include <sys/mman.h>
+#include <fcntl.h>
+
+
+#define MB (1024 * 1024)
+
+
+int fill_mem(int mb_count) {
+    int i, j;
+    char *ptr = NULL;
+    for(j = 0; j < mb_count; j++) {
+        ptr = NULL;
+        while (ptr == NULL) {
```

Comment:
There is a weird behavior on aarch64 when `mmap`ing the whole thing at once where you get a SIGSEGV before the OOM killer should send a SIGKILL in case of large memory chunks.

Reply:
Please add some code comments about this.